compilator/compilator.go

```diff
@@ -471,16 +470,21 @@ func (c *Compilator) compilePackage(pkg *model.Package) (err error) {
 			return color.GreenString("compilation-%s > %s", color.MagentaString("%s", pkg.Name), color.RedString("%s", line))
 		},
 	)
-	exitCode, container, err := c.DockerManager.RunInContainer(
-		containerName,
-		c.BaseImageName(),
-		[]string{"bash", containerScriptPath, pkg.Name, pkg.Version},
-		c.getTargetPackageSourcesDir(pkg),
-		c.getPackageCompiledTempDir(pkg),
-		c.keepContainer,
-		stdoutWriter,
-		stderrWriter,
-	)
+	mounts := map[string]string{
+		c.getTargetPackageSourcesDir(pkg): docker.ContainerInPath,
+		c.getPackageCompiledTempDir(pkg):  docker.ContainerOutPath,
+		"source_mount":                    "/var/vcap/source/",
```

Comment:
Sample volume name, after changes: `volume_fissile-cbase-fissile-2.0.1-14.g14c8518.HCF-926-tar-fails-to-compile-cf-239-dev.1-pkg-ruby-2.2.5_source_mount-8d9a1db0-45cb-4625-82d6-9d2914e9de19`
(Includes: `volume_` prefix; container name including `cbase` image with fissile version, package name; `source_mount` string; randomly generated UUID.)